test/activesupport5/decoding_test.rb

```diff
@@ -1,12 +1,14 @@
-require 'activesupport5/test_helper'
-require 'active_support/json'
-require 'active_support/time'
-require 'activesupport5/time_zone_test_helpers'
+# frozen_string_literal: true
 
 require 'oj'
 
 Oj::Rails.set_decoder()
 
+require_relative "abstract_unit"
+require "active_support/json"
+require "active_support/time"
+require_relative "time_zone_test_helpers"
```

Comment:
I think the Oj code needs to be after this or else the active support decoder is used. Have you checked?

Reply:
Ah, that's a good point! I do remember a way to check if Oj is being used, but can't find it in the docs atm. Could you maybe point me into the right direction? Then I can also add a test case for it :)

Reply:
You can turn on tracing but that doesn't help for testing. How about checking the `ActiveSupport::json_encoder` or is it `ActiveSupport.json_encoder`.

Reply:
Hmm, I can test this in the **en**coding test, but there is no `json_decoder`, so the test fails (I added it anyway, to see what I tried).